packages/prisma-client/src/types.ts

```diff
@@ -1,9 +1,10 @@
-import type { Asset as DbAsset } from "@prisma/client";
+import type { Asset as DbAsset, Project as BaseProject } from "@prisma/client";
 export { Location } from "@prisma/client";
-export type { InstanceProps, Project, User, Breakpoints } from "@prisma/client";
+export type { InstanceProps, User, Breakpoints } from "@prisma/client";
 
 export type Asset = DbAsset & {
   path: string;
   status?: "uploading" | "uploaded";
 };
+export type Project = BaseProject & { Asset: DbAsset[] };
```

Comment:
that type is probably automatically provided by prisma, no? also its not Asset, its assets now

Reply:
this has been fixed